packages/theme/composables/useConfig/index.ts

```diff
@@ -24,7 +24,7 @@ export function useConfig(): UseConfigInterface {
     Logger.debug('useConfig/load');
 
     try {
-      const { data } = await app.$vsf.$magento.api.storeConfig(params?.customQuery ?? null);
+      const { data } = await app.$vsf.$magento.api.storeConfig(params?.customQuery ?? null, params?.customHeaders ?? null);
```

Comment:
Could you do this everywhere?

```const def = {test: true};
const fun = (val = {}) => {
 console.log({...def, ...val});
}
fun(def?.doesNotExistsProp);
```
Yields 
```
{
  test: true
}
```
So the ?? null is redundant in all cases.

```suggestion
      const { data } = await app.$vsf.$magento.api.storeConfig(params?.customQuery, params?.customHeaders);
```